src/chainer.test.js

```diff
@@ -1 +1,31 @@
+'use strict';
+
 const { chainer } = require('./chainer');
+
+describe('chainer', () => {
+  it('should chain the unary functions correctly', () => {
+    const mockFunctionA = jest.fn((x) => x + 1);
+    const mockFunctionB = jest.fn((x) => x * 2);
+    const mockFunctionC = jest.fn((x) => x - 3);
+    const mockFunctionD = jest.fn((x) => x / 4);
+
+    const chain = chainer([mockFunctionA, mockFunctionB,
+      mockFunctionC, mockFunctionD]);
+
+    const input = 10;
+    const result = chain(input);
+
+    expect(mockFunctionA).toHaveBeenCalledWith(input);
```

Comment:
The current tests are tightly coupled with the implementation (i.e., the order of calls and the exact arguments). A more robust approach would be to test the output rather than the exact sequence of function invocations.